test/Microsoft.VisualStudio.LanguageServices.Razor.Test/ProjectSystem/TestProjectSystemServices.cs

```diff
@@ -87,33 +87,33 @@ public IProjectVersionedValue<IProjectSubscriptionUpdate> CreateUpdate(params Te
                 dataSourceVersions: ImmutableDictionary<NamedIdentity, IComparable>.Empty);
         }
 
-        public class TestProjectServices : IProjectServices
+        public class TestProjectServices : ProjectServices
```

Comment:
Some types we consumed changed from interface to abstract class. It only really impacts our tests, but we have to recompile because it's a binary breaking change.